scripts/linters/pre_commit_linter.py

```diff
@@ -328,32 +350,99 @@ def _get_file_extensions(file_extensions_to_lint):
     return all_file_extensions_type
 
 
-def _get_all_filepaths(input_path, input_filenames):
+def _get_filepaths_from_path(input_path):
```

Comment:
[Optional] Should we have `exclude_filepaths_patterns=[]` here and let the calling function decide what needed to be excluded?